src/Protocol/Relay.php

```diff
@@ -192,34 +188,31 @@ public static function isWantedLanguage(string $body, int $uri_id = 0, int $auth
 			}
 		}
 
-		if (empty($languages) && empty($detected) && (empty($body) || Smilies::isEmojiPost($body))) {
+		if (empty($detected) && empty($languages)) {
+			$detected = ['un'];
```

Comment:
Other location for the `UNDETERMINED_LANGUAGE` constant.

Reply:
Done